tracee-ebpf/tracee/events.go

```diff
@@ -0,0 +1,184 @@
+package tracee
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"strconv"
+	"unsafe"
+
+	"github.com/aquasecurity/tracee/tracee-ebpf/external"
+)
+
```

Comment:
Because events_pipeline.go was deleted and events.go was created it was harder for me to know if some stuff already existed in the previous _pipeline.go file. For those changes, next time, maybe we should first move the file in one commit and change the code in the other - just thought about it actually, not something I ever worried - this way we have the diff 1:1 to compare existing code (just as a 2 cents note) and review.

Reply:
Yes, I understand your point, and I actually wanted to split this into two commits at first for this exact reason.
The problem was that I also removed the `go func() {` lines which existed in the events_pipeline.go file, and which created an extra indentation. By removing it, it seemed like I changed almost every line of the file, and I think that it would have been much harder to review it like this. So I thought that creating a new file will make the review easier, as a reviewer can then open both files side by side and see the differences. Maybe I was wrong here, but my intention was to make the review easier.
Just for you to know - this patch mostly removed code, and there were only minimal changes in the logic.

Reply:
Ok, I've split the commit to few. The first commit has wrong indentation in purpose - it is now easier to review it one commit at a time. After review, we can merge all into one.